core/management/commands/import_facility_data.py

```diff
@@ -8,10 +8,9 @@
 class Command(BaseCommand):
     help = 'Import Patient Data'
 
-    def filter_columns(self, care_type, facility_type, facility_df):
+    def filter_columns(self, facility_type, facility_df):
         facility_id_column = "Facility ID" if "Facility ID" in facility_df.columns else "CMS Certification Number (CCN)"
         facility_df = facility_df.drop_duplicates()
-        print("facility_id_column", facility_id_column)
         if facility_type == 'CCN':
```

Comment:
CCN is not a facility type. Also why did this switch from care_type to facility_type

Reply:
I just removed the care_type because we weren't using care_type within this function.